### ISSUE TITLE
update bliss, URL, and new version (0.77)

### ISSUE THREAD

**Issue by dimpase** [digraphs/Digraphs#588]
See https://users.aalto.fi/~tjunttil/bliss/download.html 
```
Version 0.77 (2021-02-18):
- set_color and copy methods in the C++ API
- examples/enumerate.cc
- precise group size computation also without GNU GMP
- no more exit-causing "fatal_errors", instead a std::logic_error, std::out_of_range or std::bad_alloc is thrown
```

**Comment by james-d-mitchell** [comment]
Thanks for the issue @dimpase, we fairly heavily patched `bliss` in Digraphs, so this might not be a drop in replacement, I'm afraid.

**Comment by dimpase** [comment]
you allow using external bliss, and several Linux distros, e.g. Gentoo, now ship version 0.77.
Do you patch anything except the build system?

**Comment by james-d-mitchell** [comment]
Yes, we didn't patch the build system at all (we don't use bliss' build system), the patches are related to reducing memory allocations for the use case of a high volume of computations rather than a single long computation. They are in:

https://github.com/digraphs/bliss

**Comment by james-d-mitchell** [comment]
Just to mention that I wrote to Tommi Junttila about incorporating the changes in our version of bliss into bliss proper on the 5th of January, 2024.

**Comment by james-d-mitchell** [comment]
Haven't received any reply to my email :(
